### ISSUE TITLE
Code folding

### ISSUE THREAD

**Issue by cup** [editorconfig/editorconfig#418]
Several editors I use offer code folding:

- https://github.com/notepad-plus-plus/notepad-plus-plus
- https://github.com/rizonesoft/Notepad3
- https://github.com/ProgerXP/Notepad2e
- https://github.com/zufuliu/notepad2

It seems that EditorConfig has no way to handle the setting of enabling or
disabling code folding. Is that the case?

**Comment by greut** [comment]
_editorconfig_ offers settings to be applied to the code itself. So another developer's preference/assumption won't collide with the defined settings. Code folding doesn't affect the code, so it poses no risks of collision with someone else preferences. I don't see why you'd want to impose any kind of code folding preferences to your peers.

IMHO, it's not applicable.

**Comment by cxw42** [comment]
@cup Would you please explain what you mean by "code folding"?  Is it just the line-breaking used when code is displayed, or something else?  I have not used the feature myself.  Thanks!

**Comment by cup** [comment]
https://en.wikipedia.org/wiki/Code_folding

**Comment by xuhdev** [comment]
I agree with @greut . Code folding is a matter of personal visual preference, which settings do not intrude those of other developers on the same project. Hence it is out of the scope of EditorConfig. A more appropriate option is probably to set it in your own developer tool preference settings.

Quoted from EditorConfig website:

> EditorConfig helps maintain consistent coding styles for multiple developers working on the same project across various editors and IDEs.
